data-generation/src/main/scala/thesis/Landy.scala

```diff
@@ -54,6 +54,7 @@ class Landy(underlyingGraph: LandyAttributeGraph) extends TemporalGraph {
     }
     )
       .map(edge => if (edge.dstId == vertexId) edge.srcId else edge.dstId)
+      .distinct // Quickfix, there were duplicates returned
```

Comment:
```suggestion
      .distinct 
```
Løser ikke dette problemet på en god måte egentlig? Er det en bedre fix vi kan gjøre som ikke er en "quick fix"?